ha_cluster_exporter.go

```diff
@@ -134,7 +141,16 @@ func resetDrbdMetrics() error {
 	if err != nil {
 		return errors.Wrap(err, "failed to register DRBD disk state metric. Perhaps another exporter is already running?")
 	}
+	prometheus.Unregister(drbdRemoteDiskState)
+	drbdRemoteDiskState = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "ha_cluster_drbd_resources_remote_connection",
+			Help: "show per remote connection resource name, its role, the volume and disk_state (Diskless,Attaching, Failed, Negotiating, Inconsistent, Outdated, DUnknown, Consistent, UpToDate)",
+		}, []string{"resource_name", "peer_node_id", "peer_role", "volume", "peer_disk_state"})
```

Comment:
Is it normal that this metric re-registration has `peer_node_id` while the first one in line 81 hasn't?
BTW, we should investigate a way to avoid the duplication in all the resets functions, but that's for another PR

Reply:
@stefanotorresi  yes it is normal. peer_node_id is something more on remote concept of SBD disk. the 1st metric is only regarding the local sbd disk. The 2nd metric we need an ide because it could be multiple host having same volume

Reply:
I agree that the reset metric mechanism could be improved. At that time I couldn't find anything convenient.